Tests.Integration.Presentation.Web/Tools/External/ItSystemUsageV2Helper.cs

```diff
@@ -142,6 +142,11 @@ public static async Task<HttpResponseMessage> SendPatchGDPR(string token, Guid u
             return await HttpApi.PatchWithTokenAsync(TestEnvironment.CreateUrl($"api/v2/it-system-usages/{uuid}"), token, dto.AsPatchPayloadOfProperty(nameof(UpdateItSystemUsageRequestDTO.GDPR)));
         }
 
+        public static async Task<HttpResponseMessage> SendPatchValidity(string token, Guid uuid, ValidityWriteRequestDTO dto)
+        {
+            return await HttpApi.PatchWithTokenAsync(TestEnvironment.CreateUrl($"api/v2/it-system-usages/{uuid}"), token, dto.AsPatchPayloadOfProperty(nameof(UpdateItSystemUsageRequestDTO.GDPR)));
```

Comment:
Are you sure this works???... It will send
```
{
  "GDPR": {your_payload}
}
```

Remember that every time you do copy-paste-programming someone dies of cancer